virttest/test_setup/__init__.py

```diff
@@ -615,8 +615,9 @@ def cleanup(self):
             error_context.context("trying to deallocate hugepage memory")
             try:
                 process.system("umount %s" % self.hugepage_path)
-            except process.CmdError:
-                return
+            except process.CmdError as detail:
```

Comment:
I suggest that we can refer to the process of unmounting the huge page https://github.com/avocado-framework/avocado-vt/blob/2283147316333cf1b222905739b6c533a19ed4be/virttest/test_setup/__init__.py#L573-L576 that execute the following actions depends on the status of mounting.

Reply:
good suggestion :)